test/usb.uts

```diff
@@ -4,7 +4,7 @@
 
 = load module
 
-load_layer("usb")
+load_layer("usb", globals_dict=globals())
```

Comment:
I'm not convinced this makes things easier. `load_layer` was supposed to be a shortner for the imports (and CLI friendly).
I agree with the part of this PR that re-uses the same session, but I'm not sure why you're refactoring all of this code.

Also conflicts

Reply:
Hi, my intention wasn't the change of `load_layer` in general. I think the default usage `load_layer("whatever")` is absolutely fine in scripts or interactive sessions.

I refactored this code, because I wanted to see if more side-effects of the previous loading to `builtins` showed up. This change excludes also possibility, that previous executed tests mess up the `builtins`, which helps to get the execution behaviour of the tests on the CI machines and on local tests via `python -m scapy.tools.Utscapy` more similar. 

If you don't like the modification of the `load_layer` commands, I can revert them. I don't see a disadvantage in this change.

Reply:
Conflict resolved

Reply:
I would prefer that you revert the change made to the .uts files if they are not necessary.